packages/block-library/src/spacer/edit.js

```diff
@@ -15,6 +15,7 @@ import { withInstanceId } from '@wordpress/compose';
 const SpacerEdit = ( { attributes, isSelected, setAttributes, toggleSelection, instanceId } ) => {
 	const { height } = attributes;
 	const id = `block-spacer-height-input-${ instanceId }`;
+	const [ inputHeightValue, setInputHeightValue ] = useState( height );
```

Comment:
I'm not sure I understand why we need a sperate state value here. Why `height` is not enough?

Reply:
When the user deletes the value in the input entirely, we wanted `height` to revert to the default of `100`. That felt like the right user experience to me at the time.

It creates a situation where the input has one value `''` and the spacer has another `100`.

I was also thinking about an uncontrolled input, but then there's still a requirement to set the initial value.

Are there other ways to achieve this?

Reply:
makes sense, I didn't see this subtle difference at first. Thanks for clarifying.